x-pack/test/detection_engine_api_integration/security_and_spaces/group10/perform_bulk_action.ts

```diff
@@ -2997,5 +2997,223 @@ export default ({ getService }: FtrProviderContext): void => {
+        await fetchRule(ruleIdWithInvestigationField).expect(404);
+      });
+
+      it('should enable rules with legacy investigation fields', async () => {
+        const ruleIdWithLegacyInvestigationField = '2297be91-894c-4831-830f-b424a0ec84f0';
+        const ruleIdWithLegacyInvestigationFieldEmptyArray = '2297be91-894c-4831-830f-b424a0ec5678';
+        const ruleIdWithInvestigationField = '2297be91-894c-4831-830f-b424a0ec9102';
+
+        const { body } = await postBulkAction()
+          .send({ query: '', action: BulkActionType.enable })
+          .expect(200);
+
+        expect(body.attributes.summary).to.eql({ failed: 0, skipped: 0, succeeded: 3, total: 3 });
+
+        // Check that the updated rule is returned with the response
+        expect(
+          body.attributes.results.updated.every(
+            (returnedRule: RuleResponse) => returnedRule.enabled
+          )
+        ).to.eql(true);
+
+        // Check that the updates have been persisted
+        const { body: ruleBody } = await fetchRule(ruleIdWithLegacyInvestigationField).expect(200);
+        expect(ruleBody.enabled).to.eql(true);
+        const { body: ruleBody2 } = await fetchRule(
+          ruleIdWithLegacyInvestigationFieldEmptyArray
+        ).expect(200);
+        expect(ruleBody2.enabled).to.eql(true);
+        const { body: ruleBody3 } = await fetchRule(ruleIdWithInvestigationField).expect(200);
+        expect(ruleBody3.enabled).to.eql(true);
+      });
+
+      it('should disable rules with legacy investigation fields', async () => {
+        const ruleIdWithLegacyInvestigationField = '2297be91-894c-4831-830f-b424a0ec84f0';
+        const ruleIdWithLegacyInvestigationFieldEmptyArray = '2297be91-894c-4831-830f-b424a0ec5678';
+        const ruleIdWithInvestigationField = '2297be91-894c-4831-830f-b424a0ec9102';
+        const { body } = await postBulkAction()
+          .send({ query: '', action: BulkActionType.disable })
+          .expect(200);
+
+        expect(body.attributes.summary).to.eql({ failed: 0, skipped: 0, succeeded: 3, total: 3 });
+
+        // Check that the updated rule is returned with the response
+        expect(
+          body.attributes.results.updated.every(
+            (returnedRule: RuleResponse) => returnedRule.enabled === false
+          )
+        ).to.eql(true);
+
+        // Check that the updates have been persisted
+        const { body: ruleBody } = await fetchRule(ruleIdWithLegacyInvestigationField).expect(200);
+        expect(ruleBody.enabled).to.eql(false);
+        const { body: ruleBody2 } = await fetchRule(
+          ruleIdWithLegacyInvestigationFieldEmptyArray
+        ).expect(200);
+        expect(ruleBody2.enabled).to.eql(false);
+        const { body: ruleBody3 } = await fetchRule(ruleIdWithInvestigationField).expect(200);
+        expect(ruleBody3.enabled).to.eql(false);
+      });
+
+      it('should duplicate rules with legacy investigation fields', async () => {
```

Comment:
This one also doesn't check anything for the field specifically, although it could assert that in the duplicates the field is migrated.

Reply:
Updated tests to be more specific about what they are testing and if SO migration is inteded.